packages/hs-react-ui/src/components/RangeSlider/RangeSlider.tsx

```diff
@@ -0,0 +1,312 @@
+import React, { useEffect, useRef, useState } from 'react';
+import styled from 'styled-components';
+import debounce from 'lodash/debounce';
+
+import { useSpring, a } from 'react-spring';
+import { useDrag } from 'react-use-gesture';
+import useMeasure from 'react-use-measure';
+import { ResizeObserver } from '@juggle/resize-observer';
+
+import fonts from '../../enums/fonts';
+import colors from '../../enums/colors';
+import { clamp } from '../../utils/math';
+
+import {
+  valueProp,
+  containerProps,
+  handleProps,
+  handleLabelProps,
+  RangeSliderProps,
+  selectedRangeProps,
+  domainLabelProps,
+} from './types';
+
+export const Container = styled.div`
+  ${({ showDomainLabels, hasHandleLabels, disabled, beingDragged = false }: containerProps) => `
+    position: relative;
+    height: 1rem;
+    width: 100%;
+
+    ${fonts.body}
+
+    user-select: none;
+
+    ${beingDragged ? 'cursor: grabbing;' : ''}
+
+    transition: filter .1s;
+
+    ${
+      disabled
+        ? `
+      filter: grayscale(1) contrast(.5) brightness(1.2);
+      pointer-events: none;
+    `
+        : ''
+    }
```

Comment:
Not sure if prettier went ham, but there's a lot of unneeded line breaks here.

Reply:
It really went crazy